### PR TITLE
Jibola.ola

### PR DESCRIPTION
**Pull request by JibolaPeace**
Kindly check my PR and approve. Thanks

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <title>Javascript 101</title>
-        <script src="scripts/oshomo.oforomeh-v2.js"></script>
+                 <script src = "jibola.ola.v2.js"></script>
```
Comment (hoshomoh) [inline index.html:6]: Keep my script. We can have multiple script on a page. You can add yours in the next line.

File: scripts/jibola.ola.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var firstName = "jibola";
```
Comment (hoshomoh) [inline scripts/jibola.ola.js:1]: Do not use var. Either you user `let` or `const`

File: index.html (hunk 1 above)
Comment (JibolaPeace) [reply index.html:6]: Changes done.

Comment (hoshomoh) [reply index.html:6]: I don't see it.

Comment (JibolaPeace) [reply index.html]: Changes made and empty files removed.

File: index.html [hunk 3]
```diff
@@ -4,6 +4,7 @@
         <meta charset="utf-8">
         <title>Javascript 101</title>
         <script src="scripts/oshomo.oforomeh-v2.js"></script>
+        <script src ="jibola.ola.v2.js"></script>
```
Comment (hoshomoh) [inline index.html:7]: This path is not correct

File: scripts/jibola.ola.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+const firstName = "jibola";
+const lastName = "ola";
+const fullName = `${firstName} ${lastName}`;
+const age = 22;
+const salaryExpectationn = 55000.00;
```
Comment (hoshomoh) [inline scripts/jibola.ola.js:5]:
```suggestion
const salaryExpectation = 55000.00;
```

File: index.html (hunk 3 above)
Comment (JibolaPeace) [reply index.html]: Corrected it.

**Review by hoshomoh — APPROVED** [review]

**Pull request merged**